### PR TITLE
Hello-Microverse-First-Pull-Request

### PR DESCRIPTION
**Pull request by koomekevo**

### CODE REVIEW

File: hello-microverse.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (carlos-ssh) [inline hello-microverse.html:1]: - Provide a description in your Pull Request about the features implemented.

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
```
Comment (carlos-ssh) [inline README.md]: - Provide a README fully described you can use the [template](https://github.com/microverseinc/readme-template) provided by Microverse.

File: README.md [hunk 3]
```diff
@@ -1 +1,5 @@
...
+This project aims to showcase Github flow and Linters.
+# built with
+HTML
+CSS
```
Comment (carlos-ssh) [inline README.md]: - Add in your ``.gitignore`` file your ``test.md`` file and ``node_modules`` directory.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @koomekevo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

changesrequested




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,28 @@
...
+## Built With
+
+- Html
+- Css
+- Github
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (Mihndim2020) [inline README.md]: - [ ] This should be steps on how to clone this project locally(on your local machine) and how to run it. Please kindly fix it.

File: README.md [hunk 5]
```diff
@@ -1 +1,28 @@
...
+
+
+## Authors
+
+👤 **Kevin Koome**
+
+- GitHub: [@githubhandle](https://github.com/koomekevo)
+- Twitter: [@twitterhandle](https://twitter.com/koomekevo)
+
+/in/linked
```
Comment (Mihndim2020) [inline README.md]: - [ ] Please kindly remove this as it make your readme dirty. Visitors are potential recruiters need to see that you page attention to details.

File: README.md [hunk 6]
```diff
@@ -1 +1,28 @@
...
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+## Authors
+
+👤 **Kevin Koome**
+
+- GitHub: [@githubhandle](https://github.com/koomekevo)
+- Twitter: [@twitterhandle](https://twitter.com/koomekevo)
+
```
Comment (Mihndim2020) [inline README.md:21]: - [ ] Kindly update you readme file with the information in the image below. See [template]([url](https://github.com/microverseinc/readme-template))
koom

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @koomekevo , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments below under the review._

Please do consider, as requested by the previews reviewer:
- [ ] Pull Request should have a short summary describing introduced changes in more details i.e what you have done in this particular milestone(eg; created an index.html, created style.css file etc).

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Kevin Mbae, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**